azkaban-execserver/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -125,6 +130,43 @@ public AzkabanExecutorServer(Props props) throws Exception {
     logger.info("Azkaban Executor Server started on port " + portNumber);
   }
 
+  /**
+   * Configure Metric Reporting as per azkaban.properties settings
+   */
+  private void configureMetricReports() {
+    Props props = getAzkabanProps();
+    if (props != null && props.getBoolean("executor.metric.reports", false)) {
+      logger.info("Starting to configure Metric Reports");
+      MetricReportManager metricManager = MetricReportManager.getInstance();
+      IMetricEmitter metricEmitter = new InMemoryMetricEmitter(props);
+      metricManager.addMetricEmitter(metricEmitter);
+
+      logger.info("Adding number of failed flow metric");
```

Comment:
Sorry, should have commented on this sooner.  Can we use constants for "executor.metric.interval.default" and "executor.metric.interval"?